src/Rule/DateTimeHandler.php

```diff
@@ -32,13 +32,23 @@ public function validate(mixed $value, object $rule, ValidationContext $context)
         }
         \DateTime::createFromFormat($rule->getFormat(), (string)$value);
 
+        $errors = \DateTime::getLastErrors();
         // Before PHP 8.2 may return array instead of false (see https://github.com/php/php-src/issues/9431).
-        $errors = \DateTime::getLastErrors() ?: [ 'error_count' => 0, 'warning_count' => 0 ];
-        if ($errors['error_count'] !== 0  ||  $errors['warning_count'] !== 0) {
-            $result->addError($rule->getMessage(), [
-                'attribute' => $context->getTranslatedAttribute(),
-                'value' => $value,
-            ]);
+        /** @infection-ignore-all */
```

Comment:
Can you add tests instead of ignore?

Reply:
I make a little refactoring. It is help remove mutants. If this OK for you, then I will merge this PR.

Reply:
if you need to kill only in version 81, then ok.

Reply:
Good catch 👍. I changed version for mutation testing to 8.3. And fix mutants.